src/commcare_cloud/ansible/roles/nodejs/tasks/main.yml

```diff
@@ -31,7 +31,7 @@
     - npm
 
 - name: Update Node
-  command: 'n 16.19.1'
+  command: 'n 20.11.1'
```

Comment:
Is it possible to have this install the most recent patch of v20 rather than this specific version?

Reply:
I think `n 20.11.*` should do the trick. 
I can look into it but won't be able to spend a lot of time in figuring it if it turns out to be complex.

Reply:
Seems like for node we can just pass in minor version and it should work and for npm version * worked fine.